src/pipeline_configuration_spec.py

```diff
@@ -40,11 +40,12 @@ class RapidProTarget:
 @dataclass
 class PipelineConfiguration:
     pipeline_name: str
+    test_participant_uids: []
```

Comment:
why do we need both test participant uids and filter_test_participants? Can we just have test_participant_uids: [] | None and only filter when the ids are specified?

Reply:
updated thanks.